weighted_levenshtein/__init__.py

```diff
@@ -1 +1,2 @@
-from clev import *
+from .clev import *
+
```

Comment:
nit: Remove blank newline at the EOF (the last character of the file should still be a newline, just not the last two characters)